example_audit.rules

```diff
@@ -78,49 +87,112 @@
 -w /var/spool/cron/root -p wa -k cron
-
-# Record changes to init systems config
+-w /var/spool/atspool
+-w /etc/at.allow
+-w /etc/at.deny
+ 
+# Modify startup
 -w /etc/rc.d/init.d/ -p wa -k init
 -w /sbin/init -p wa -k init
 -w /etc/inittab -p wa -k init
--w /etc/systemd -p wa -k init
+-w /etc/systemd/ -p wa -k init
+-w /etc/init.d/ -p wa -k init
 
-# Record changes to access control config
+# Modify PAM
 -w /etc/pam.d -p wa -k pam
--w /etc/security -p wa -k pam
--w /lib/security -p wa -k pam
--w /etc/sudoers -p wa -k user
-
-# Record changes to OpenSSH daemon config
--w /etc/sshd -p wa -k sshd
-
-# Record changes to user/passwords
+-w /etc/security/ -p wa -k pam
+-w /lib/security/ -p wa -k pam
+ 
+# Modify SSHD
+-w /etc/sshd/ -p wa -k sshd
+ 
+# Users
 -w /etc/group -p wa -k user
 -w /etc/passwd -p wa -k user
 -w /etc/gshadow -p wa -k user
 -w /etc/shadow -p wa -k user
 -w /etc/security/opasswd -p wa -k user
-
-# Change clock
+-w /etc/sudoers -p wa -k user
+ 
+# Time changes
 -a always,exit -F arch=b32 -S adjtimex -S settimeofday -k time-change
 -a always,exit -F arch=b64 -S adjtimex -S settimeofday -k time-change
 -w /etc/localtime -p wa -k time-change
-
-# Linux kernel module loading
+ 
+# Module loading
 -a exit,always -F arch=b64 -S init_module -k module
 -a exit,always -F arch=b32 -S init_module -k module
-
-# kexec syscall (re-exec kernel)
+ 
+# Swapping kernel via kexec
 -a exit,always -F arch=b64 -S kexec_load -k kexec
 -a exit,always -F arch=b32 -S kexec_load -k kexec
 
-# Record changes to system binaries
--w /usr/bin -p wa -k binaries
--w /bin -p wa -k binaries
--w /usr/sbin -p wa -k binaries
--w /sbin -p wa -k binaries
--w /usr/local/bin -p wa -k binaries
--w /usr/local/sbin -p wa -k binaries
+# Attaching to processes
+-a exit,always -F arch=b64 -S ptrace -k trace
+-a exit,always -F arch=b32 -S ptrace -k trace
+
+# Changes to binaries (common locations)
+-w /usr/bin/ -p wa -k binaries
+-w /bin/ -p wa -k binaries
+-w /usr/sbin/ -p wa -k binaries
+-w /sbin/ -p wa -k binaries
+-w /usr/local/bin/ -p wa -k binaries
+-w /usr/local/sbin/ -p wa -k binaries
+
+# Changes to libraries (common locations)
+-w /lib/ -p wa -k binaries
+-w /lib32/ -p wa -k binaries
+-w /lib64/ -p wa -k binaries
+-w /usr/lib/ -p wa -k binaries
+-w /usr/lib32/ -p wa -k binaries
+-w /usr/lib64/ -p wa -k binaries
+-w /usr/local/lib/ -p wa -k binaries
+-w /usr/local/lib32/ -p wa -k binaries
+-w /usr/local/lib64/ -p wa -k binaries
+-w /usr/libexec/ -p wa -k binaries 
+-w /usr/local/libexec/ -p wa -k binaries
+
+# Important system files
+-w /etc/sysctl.conf -p wa -k binaries
+-w /etc/modprobe.conf
+-w /etc/modprobe.conf.local
+-w /etc/modprobe.d/ -p wa -k binaries
+-w /etc/modules-load.d/ -p wa -k binaries
+
+# Kernel modules
+-w /lib/modules/ -p wa -k binaries
+
+# Libc preload and linking configuration
+-w /etc/ld.so.conf -p wa -k binaries
+-w /etc/ld.so.conf.d -p wa -k binaries
+-w /etc/ld.so.preload -p wa -k binaries
+
+# System-wide shell profiles
```

Comment:
i dont know if i would log shell env changes, not super useful, slows down syscall entry